.circleci/config.yml

```diff
@@ -25,7 +28,19 @@ jobs:
   account:
     <<: *steps
   block:
-    <<: *steps
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: 
+          name: Installing dependencies
+          command: cd ~/project/packages/${CIRCLE_JOB} && npm install
```

Comment:
For this PR, `Block` won't use Lerna for linking packages. So in both CIs, we need to explicitly `npm install` for `Block` package.